search_log.go

```diff
@@ -136,34 +177,45 @@ func readLine(reader *bufio.Reader) (string, error) {
 	return string(line), nil
 }
 
-func readLastLine(file *os.File) string {
-	var line []byte
-	var cursor int64
-	stat, _ := file.Stat()
-	filesize := stat.Size()
+// Read lines from the end of a file
+// endCursor initial value should be the filesize
+func readLastLines(file *os.File, endCursor int64) string {
```

Comment:
`readLastLines` look like maybe returns multiple lines, but the return parameter is `string`, should be `[]string`?

Reply:
I need to get read bytes by `len(string)` in its caller method, return `[]string` will miss the length info, else I need to add an extra `readSize` return value. Which way is better?

Reply:
I prefer `lines []string, size int`, it's more clear

Reply:
ok!

Reply:
Done!